README.md

```diff
@@ -2,9 +2,9 @@
 
 <img align="right" width="200" height="200" src="./docs/fia-logo.svg" alt="Feature Info Agent Logo">
 
-This Feature Info Agent (FIA) acts as a single access point for [TWA Visualisations](https://github.com/cambridge-cares/TheWorldAvatar/wiki/TWA-Visualisations) to query for both meta and time series data of an individual feature (i.e. a single geographical location) so that it can then be displayed within the side panel of the visualisation.
+This Feature Info Agent (FIA) acts as a single access point for [TWA viz](https://github.com/TheWorldAvatar/viz) instances to query for both meta and time series data of an individual feature (i.e. a single geographical location) so that it can then be displayed within the side panel of the visualisation.
 
-Please see the [CHANGELOG](./CHANGELOG.md) file for details on recent changes; the latest available image of the FIA can be determined by viewing its [GitHub package page](https://github.com/cambridge-cares/TheWorldAvatar/pkgs/container/feature-info-agent).
+Please see the [CHANGELOG](./CHANGELOG.md) file for details on recent changes; the latest available image of the FIA can be determined by viewing its [GitHub package page](https://github.com/TheWorldAvatar/pkgs/container/feature-info-agent).
```

Comment:
I'm not sure that package repo URL is correct. Also there isn't a image deployed to it yet.
Once an image has been deployed we'll know the URL for certain.